analysisservice/src/main/java/nu/yona/server/AnalysisServiceApplication.java

```diff
@@ -51,6 +51,6 @@ public Docket yonaApi()
 	@Bean
 	public CacheManager cacheManager()
 	{
-		return new ConcurrentMapCacheManager("goalConflictMessages");
+		return new ConcurrentMapCacheManager("goalConflictMessages", "usersAnonymized");
```

Comment:
To prevent the duplication of information, I propose to define the cache manager in ``CoreConfiguration``.